pixeltable/func/expr_template_function.py

```diff
@@ -78,6 +78,18 @@ def instantiate(self, args: Sequence[Any], kwargs: dict[str, Any]) -> 'exprs.Exp
         result = result.substitute(arg_exprs)
         return result
 
+    def call_return_type(self, bound_args: dict[str, 'exprs.Expr']) -> ts.ColumnType:
```

Comment:
Explain why this needs to be special-cased.

Reply:
Done.